.github/workflows/ci.yml

```diff
@@ -165,6 +167,25 @@ jobs:
       run: ctest --preset macos-ci-benchmark
 
 
+  build_documentation_doxygen:
+    name: Documentation Doxygen
+    needs: changed
+    if: ${{ needs.changed.outputs.game_code == 'true' || needs.changed.outputs.unit_tests == 'true' || needs.changed.outputs.cmake_files == 'true' || needs.changed.outputs.ci_config == 'true' }}
```

Comment:
Does this trigger if only the doxygen configuration was changed? I don't think that was added to `changed`.

Reply:
You're right. It wasn't. It should be now.